doc/tags

```diff
@@ -0,0 +1,28 @@
+snipe-B	snipe.txt	/*snipe-B*
```

Comment:
The latest master should have this gitignored. We can add `tags` to `.gitignore` here; I'm not sure where it came from.